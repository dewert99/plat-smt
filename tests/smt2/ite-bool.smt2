(set-logic QF_UF)
(declare-const a Bool)
(declare-const b Bool)
(declare-const c Bool)
(declare-const d Bool)

(assert (ite a b c))
(assert (= d (ite a c b)))
(assert (or a d))
(assert (xor b c))
(check-sat-assuming ((not c)))
(get-model)
(check-sat-assuming (c))