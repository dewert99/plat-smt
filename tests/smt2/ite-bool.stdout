sat
(
 (define-fun a () Bool true)
 (define-fun b () Bool true)
 (define-fun c () Bool false)
 (define-fun d () Bool false)
)
unsat
